### DISCUSSION TITLE
How to acquire moose soon

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#17658] [Q&A General] [answered]
Hi moose experts and users,

This is a general question. Although I roughly look through the get started section including examples, tutorials as well as the complete syntax, and try my best to understand them, there are still much confusion in my head and I am not able to write complete code for my own model if there are no any example files for reference. I have no idea what syntax I should use even if I go through complete syntax. Does anyone share any good experiences or tips with me? Thanks for your help in advance.

**Comment by aeslaughter** [comment, marked as answer]
I suggest you start here: https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/index.html

This includes a step-by-step guide for creating a MOOSE-based application. If you read it completely and do the steps yourself you will begin to understand the process. It would also be worth watching the video on the homepage.
